src/main/java/com/mraof/minestuck/data/tag/MinestuckBlockTagsProvider.java

```diff
@@ -109,16 +109,10 @@ protected void addTags()
 		tag(MINEABLE_WITH_AXE).add(DEAD_LOG.get(), DEAD_WOOD.get(), DEAD_PLANKS.get());
 		tag(MINEABLE_WITH_AXE).add(PETRIFIED_LOG.get(), PETRIFIED_WOOD.get());
 		tag(MINEABLE_WITH_AXE).add(TREATED_PLANKS.get());
-		tag(MINEABLE_WITH_AXE).add(GLOWING_BOOKSHELF.get(), FROST_BOOKSHELF.get(), RAINBOW_BOOKSHELF.get(), END_BOOKSHELF.get(), DEAD_BOOKSHELF.get(), TREATED_BOOKSHELF.get());
-		tag(MINEABLE_WITH_AXE).add(GLOWING_LADDER.get(), FROST_LADDER.get(), RAINBOW_LADDER.get(), END_LADDER.get(), DEAD_LADDER.get(), TREATED_LADDER.get());
```

Comment:
It appears that when you did the Merge Conflict process, you accidentally chose to delete work that was added by Hadean's recent Commit

Reply:
It happened elsewhere in this file and it caused the generated block tag files associated with these lines of code to disappear as well when you did runData